### PR TITLE
Test compilation of all csx files

### LINKED ISSUE #109: check compilation of all csx files under artifacts\files
From https://github.com/adamralph/simple-targets-csx/pull/108#pullrequestreview-73571918

Currently we have to remember to add each new file manually. The horror.

### PR DESCRIPTION
**Pull request by blairconrad**
Fixes #109.

### CODE REVIEW

File: build.cmd [hunk 1]
```diff
@@ -49,12 +49,9 @@ set RUNNER=".\packages\Microsoft.Net.Compilers.%CSI_VERSION%\tools\csi.exe"
...
 @echo On
 
-%RUNNER% .\artifacts\files\internal\runner.csx || goto :error
-%RUNNER% .\artifacts\files\internal\target-runner.csx || goto :error
-%RUNNER% .\artifacts\files\internal\target-runner-options.csx || goto :error
-%RUNNER% .\artifacts\files\internal\util.csx || goto :error
-%RUNNER% .\artifacts\files\simple-targets-target.csx || goto :error
-%RUNNER% .\artifacts\files\simple-targets.csx || goto :error
+for /R .\artifacts\files\ %%f in (*.csx) do (
+      %RUNNER% "%%f" || goto :error
```
Comment (adamralph) [inline build.cmd]: Does the indent have to be this big, or can it be 2 chars, to match the rest of the file (see the `if` statements further up)?

**Review by adamralph — COMMENTED** [review]
😍 

image

Just left one lovely nitpick for you.

File: build.cmd (hunk 1 above)
Comment (blairconrad) [reply build.cmd]: Odd, I thought I'd conformed. I'd blame VS Code, saying that it re-indented for me, but when I reduced to 2 spaces, it left the file alone. Something foolish on my part, I think. Amending.

**Review by adamralph — APPROVED** [review]
Thanks @blairconrad!

**Comment by blairconrad** [comment]
Thank you.

**Pull request merged**